scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc

```diff
@@ -34,12 +34,11 @@ include::modules/ztp-enabling-assisted-installer-service-on-bare-metal.adoc[leve
 
 include::modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc[leveloffset=+1]
 
+include::modules/ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries.adoc[leveloffset=+1]
 
 include::modules/ztp-preparing-the-hub-cluster-for-ztp.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc#ztp-preparing-the-ztp-git-repository_ztp-preparing-the-hub-cluster[Preparing the GitOps ZTP site configuration repository]
```

Comment:
Why have you removed this xref?

Reply:
@ktothill I've made this change and merged a PR which fixes the build error. Please update your local copy :)

Reply:
Updated as suggested.

Reply:
Did you rebase your local copy like I suggested? you've reintroduced the error I fixed.